### PR TITLE
Implement data translator for response package

### PR DESCRIPTION
**Pull request by mjbrisebois**
Replaces handling of irregular response format with a standardized data translation library

- [@holo-host/data-translator](https://www.npmjs.com/package/@holo-host/data-translator)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -79,3 +79,9 @@ publish-docs:
 	@echo "\nForce push to gh-pages"
 	git push -f origin gh-pages
 	git checkout $(CURRENT_BRANCH)
+
+
+use-local-hhdt:
+	npm uninstall --save @holo-host/data-translator; npm install --save ../data-translator-js
+use-npm-hhdt:
+	npm uninstall --save @holo-host/data-translator; npm install --save @holo-host/data-translator
```
Comment (robbiecarlton) [inline Makefile:87]: do we still need this? Looks like temporary tools for while you were working on the library.

**Review by robbiecarlton — APPROVED** [review]
One small change, at your discretion. If you want to keep that stuff there I'm not worried about it.

**Review by robbiecarlton — CHANGES_REQUESTED** [review]
I get the same failing tests as in https://github.com/Holo-Host/chaperone/pull/69

**Review by robbiecarlton — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (mjbrisebois) [reply Makefile:87]: No, these are usable by any dev who clones repos into the same directory (eg. `src` or `projects`) which I think is a pretty standard assumption.  Also, the only time a dev would use this is if they are contributing to both repos at the same time and they can quite easily arrange their directories to work this way.

**Pull request merged**